### ISSUE TITLE
No vimPlugins on unstable channel?

### ISSUE THREAD

**Issue by rollschild** [NixOS/nixpkgs#125898]
## Issue description
Hi,

I just did a fresh installation of 21.05. After installation I switched to both `nixos-unstable` and `nixpkgs-unstable`. After `nixos-rebuild switch` I went into my NeoVim but got an error saying it could not find `gruvbox`. So I did a search and just realized that the entire `vimPlugins` is not available on [unstable](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&query=vimPlugins) at all? Could someone please help?

## Technical details

```
these paths will be fetched (0.05 MiB download, 0.28 MiB unpacked):
  /nix/store/4cbfz95883qj70siyfj0jnal71hdrpsy-bash-interactive-4.4-p23-dev
copying path '/nix/store/4cbfz95883qj70siyfj0jnal71hdrpsy-bash-interactive-4.4-p23-dev' from 'https://cache.nixos.org'...
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.40, NixOS, 21.11pre293730.1ca6b0a0cc3 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.12`
 - channels(root): `"nixos-21.11pre293730.1ca6b0a0cc3, nixpkgs-21.11pre293858.141b2b111d7"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by Artturin** [comment]
what happens when you `nix-shell -p vimPlugins.gruvbox`

**Comment by teto** [comment]
I think you suffer from an issue in the generation of your init.vim. It's fixed on master and probably on recent nixos-unstable too. 
The plugins are available, just not loaded by neovim. Make sure to update everything (check https://github.com/NixOS/nixpkgs/pull/124785 for more info).

**Comment by rollschild** [comment]
> The plugins are available, just not loaded by neovim. Make sure to update everything (check #124785 for more info).

Thank you for the reply. But if you go to [search NixOS packages](https://search.nixos.org/packages) and search for "vimPlugins" on unstable, what do you see? Unless I'm missing something but I got an empty search results. Doesn't that mean this package is not available on unstable? Please see the attached screenshot. Thanks! 
<img width="1250" alt="Screen Shot 2021-06-07 at 2 31 43 PM" src="https://user-images.githubusercontent.com/6456628/121070967-58e30d00-c79d-11eb-8635-3f2be3643226.png">

**Comment by Artturin** [comment]
theres issues with the search sometimes
`nix search` still shows the plugins

**Comment by teto** [comment]
just `cat $(readlink -f $(which nvim))` and follow the generated config (`-u /nix/store/xxxx-init.vim`) and  it will show you the plugins it loads (or within vim type `:scriptnames`)

**Comment by rollschild** [comment]
Okay. I figured it out. It turned out that in addition to the issue @teto mentioned, there was also an error in my `customRC` for NeoVim. So somehow it no longer recognizes 
```
colorscheme gruvbox
```
So I had to delete this line and add one more line:
```
autocmd vimenter * ++nested colorscheme gruvbox
```
to `customRC`, based on the official [installation guide](https://github.com/morhetz/gruvbox/wiki/Installation), and it got rid of the error. 

Thanks for all the help! Closing this issue for now.
